app/views/newflow/student_signup/student_signup_form.html.erb

```diff
@@ -134,6 +134,17 @@
                 <% end  %>
 
                 <div class="content control-group checkboxes-section">
+
+                    <% if is_BRI_book_adopter? %>
```

Comment:
Students don't need to see the Title I question - only educators go through this flow.
Sorry, this wasn't clear on the card.

Reply:
Oh! 😣 I'm on it.